src/Pages/Login/Login.js

```diff
@@ -1,9 +1,100 @@
 import React, { Component } from 'react';
+import './Login.scss';
+import { LoginApi } from '../../config';
+import { withRouter } from 'react-router-dom';
 
 class Login extends Component {
+  constructor() {
+    super();
+    this.state = { id: '', pw: '' };
+  }
+
+  goToMain = () => {
+    this.props.history.push('/');
+  };
+
+  handleInput = e => {
+    const { value } = e.target;
+    this.setState({
+      [e.target.name]: value,
+    });
+  };
+
+  handleClick = e => {
+    this.state.id.length >= 5 && this.state.pw.length >= 8
+      ? this.goToMain()
+      : alert('로그인실패');
+  };
```

Comment:
삼항연산자 잘 활용하셨네요!! :) 
비구조화할당까지 하면 완벽! 
```suggestion

  handleClick = e => {
    const { id, pw } = this.state;
    id.length >= 5 && pw.length >= 8
      ? this.goToMain()
      : alert('로그인실패');
  };
```

Reply:
감사합니당~

Reply:
리뷰 반영해 주세요!